server/settings_test.go

```diff
@@ -192,6 +195,51 @@ func TestSettings(t *testing.T) {
 				assert.True(t, resg.Payload.Settings.SttEnabled)
 			})
 
+			// Verify Failed checks alerts in Alertmanager
+			t.Run("VerifyFailedChecksInAlertmanager", func(t *testing.T) {
+				defer restoreDefaults(t)
+				// Enabling STT
+				res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
+					Body: server.ChangeSettingsBody{
+						EnableStt: true,
+					},
+					Context: pmmapitests.Context,
+				})
+				require.NoError(t, err)
+				assert.True(t, res.Payload.Settings.TelemetryEnabled)
+
+				activeAlerts := true
+				silencedAlerts := false
+				alertsCount := 0
```

Comment:
https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style (but please read the whole piece).

(I'm not sure how I lost that comment in my previous review; "_but again_, read the whole piece" in other comment referred to this comment. My apologies.)